src/Controls/samples/Controls.Sample.Droid/MainActivity.cs

```diff
@@ -10,7 +10,10 @@ public class MainActivity : MauiAppCompatActivity
 	{
 		protected override void OnCreate(Bundle savedInstanceState)
 		{
+			Microsoft.Maui.Controls.Compatibility.Forms.Init(this, savedInstanceState);
```

Comment:
Even though this is still compat, it is actually needed because fonts require the use of some members of `Device` which are not yet available in core. We can probably remove this in a new PR and do it right-er.